ucsb-cs156-w24/proj-courses-w24-5pm-1
# Associated Courses aren't deleted with Schedule

Made some changes to backend to call delete on all associated courses on delete of Personal  Schedule

Note: Frontend working, jacoco test full coverage and passing

To test:

1. Create a Schedule
2. Add some courses to that schedule
3. Delete Schedule
4. See if associated Courses were deleted

dokku: https://proj-courses-bravo-luis-dev.dokku-05.cs.ucsb.edu

closes #6
